### PR TITLE
[test] Add test for issue#6113

### PR DESCRIPTION
**Pull request by wanghongjuan**
- Add test for issue #6113
- This test is failed on failed version 0.24.3, pass on fixed version 0.27.4

### CODE REVIEW

File: test/sanity/issue6113-mac-create-crash/test.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+chrome_options = Options()
+chrome_options.add_argument("nwapp=" + os.path.dirname(os.path.abspath(__file__)))
+
+driver = webdriver.Chrome(executable_path=os.environ['CHROMEDRIVER'], chrome_options=chrome_options)
+driver.implicitly_wait(2)
+try:
+    handles = driver.window_handles
+    print handles
+    driver.switch_to.window(handles[1])
```
Comment (rogerwang) [inline test/sanity/issue6113-mac-create-crash/test.py]: There is a race condition here: the number of windows/handles may not be 2 when this step is reached. consider using wait_for_window_handles in nw_utils.

Comment (wanghongjuan) [reply test/sanity/issue6113-mac-create-crash/test.py]: Added wait_window_handles in test.py

Comment (rogerwang) [reply test/sanity/issue6113-mac-create-crash/test.py]: > Added wait_window_handles in test.py

I don't see any update in this PR.

File: test/sanity/issue6113-mac-create-crash/test.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from nw_util import *
+
+chrome_options = Options()
+chrome_options.add_argument("nwapp=" + os.path.dirname(os.path.abspath(__file__)))
+
+driver = webdriver.Chrome(executable_path=os.environ['CHROMEDRIVER'], chrome_options=chrome_options)
+driver.implicitly_wait(2)
+try:
+    wait_window_handles
```
Comment (wanghongjuan) [inline test/sanity/issue6113-mac-create-crash/test.py]: @rogerwang Added line 13, 14 and 22, please see here, maybe the page doesn't refresh.

Comment (rogerwang) [reply test/sanity/issue6113-mac-create-crash/test.py]: does the function call need arguments?

Comment (wanghongjuan) [reply test/sanity/issue6113-mac-create-crash/test.py]: Oops, updated it.

**Comment by rogerwang** [comment]
@wanghongjuan the case failed to run: 

http://buildbot-master.nwjs.io:8010/builders/nw28_sdk_mac64/builds/34/steps/test_release/logs/stdio

Please add another PR to fix it soon.

**Comment by wanghongjuan** [comment]
Submitted a fixed PR: #6431

**Pull request merged**